### DISCUSSION TITLE
Could we use tenant per schema?

### DISCUSSION THREAD

**Discussion by aswanee** [abpframework/abp#11953] [Ideas]
The Decomintation defined two approaches to implement the multi-tenancy:
Separate DB or One DB one schema and TenanID for each tenant 
could we add schema (tenant name) as the third tenant approach ??
What I mean to group each Tenant Collection or Table under one DB with different schema

**Comment by maliming** [comment]
You can check the document first.

https://docs.abp.io/en/abp/latest/Entity-Framework-Core-Migrations#separating-host-tenant-database-schemas
